node/tests/k8st/infra/calico-kdd.yaml

```diff
@@ -29,6 +29,13 @@ metadata:
   name: calico-node
   namespace: kube-system
 ---
+# Source: calico/templates/calico-node.yaml
```

Comment:
I was expecting to see this change in `manifests/calico.yaml` and other files in the `manifests/` directory as well.

You probably need to update `charts/calico/calico-node.yaml` to include this serviceaccount, and then run `make generate`